packages/codemods/src/codemods/v6.x.x/replaceComponentSvgs/replaceComponentSvgs.ts

```diff
@@ -111,15 +122,20 @@ export default async function transform(file: FileInfo, api: API) {
 
         // The absolute path to the new file
         const outputPath = path.join(
-          path.dirname(filePath),
```

Comment:
Reading this code in isolation it was hard to know what `filePath` was. Path to the file we're parsing? Path to the svg we found in that file? Path to the file we're going to generate?

This change hopefully makes it more clear that it's the path to the svg file the code is importing